src/components/Table.tsx

```diff
@@ -22,7 +22,7 @@ interface TableColumn {
 	icon?: string;
 	label?: string | JSX.Element;
 	sortable?: boolean | TableSortFunction;
-	render?: (value: any) => string | JSX.Element;
+	render?: (value: any, context: any) => string | JSX.Element;
```

Comment:
NIT: how about naming this as `dataItem` or `row`?

Reply:
I'll change it to row